lib/code_corps_web/views/category_view.ex

```diff
@@ -1,9 +1,13 @@
 defmodule CodeCorpsWeb.CategoryView do
   @moduledoc false
   use CodeCorpsWeb, :view
-  use JaSerializer.PhoenixView
+  use JSONAPI.View
 
-  attributes [:name, :slug, :description]
+  def render("show.json-api", %{ data: category, conn: conn, params: params }) do
+    __MODULE__.show(category, conn, params)
```

Comment:
`params` is an unused arg to the `show` function.  Candidate for a PR to the lib.